packages/edit-site/src/components/sidebar/spacing-panel.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	__experimentalBoxControl as BoxControl,
+	PanelBody,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useEditorFeature } from '../editor/utils';
+
+export function useHasSpacingPanel( { supports } ) {
+	return supports.includes( 'paddingBottom' );
+}
+
+function filterUnitsWithSettings( settings = [], units = [] ) {
+	return units.filter( ( unit ) => {
+		return settings.includes( unit.value );
+	} );
+}
+
+function useCustomUnits( units ) {
+	const availableUnits = useEditorFeature( 'spacing.units' );
```

Comment:
Besides the path we need to pass the context name to useEditorFeature here otherwise it will always refer to the global context.